client/src/app/shared/video/abstract-video-list.ts

```diff
@@ -11,12 +12,15 @@ import { AuthService } from '../../core/auth'
 import { ComponentPagination } from '../rest/component-pagination.model'
 import { SortField } from './sort-field.type'
 import { Video } from './video.model'
+import { FeedFormat } from '../../../../../shared'
+import { VideoFeedComponent } from '@app/shared/video/video-feed.component'
 
 export abstract class AbstractVideoList implements OnInit, OnDestroy {
   private static LINES_PER_PAGE = 4
 
   @ViewChild('videosElement') videosElement: ElementRef
   @ViewChild(InfiniteScrollerDirective) infiniteScroller: InfiniteScrollerDirective
+  @ViewChild(VideoFeedComponent) videoFeed
```

Comment:
Do you really need the `videoFeed` variable?

Reply:
Not anymore, I just tried direct access, without much result…